lib/dotenv/beefy/railtie.rb

```diff
@@ -6,15 +6,36 @@ module Beefy
     class Railtie < Rails::Railtie
       config.before_configuration { load_environments }
 
+      # Load environment dotfiles in the following order (e.g. if in "test" environment on "darwin")
+      # 1. .env
+      # 2. .env.local
+      # 3. .env.test
+      # 4. .env.darwin
+      # 5. .env.test.local
+      # 6. .env.darwin.local
+      # 
+      # The order matters, because files loaded later will override values set previously.
+      # .env.local will override any similar keys/values found in .env
+      # .env.test.local will override any similar values found in .env.test, and .env, etc.
```

Comment:
This seems totally reasonable to me. Do you think it's worth an update to the README?

Reply:
Oh, absolutely!